text/src/autogluon/text/text_prediction/text_prediction.py

```diff
@@ -431,11 +427,19 @@ def fit(cls, train_data,
                                      .format(time_limits))
         if num_trials is None:
             num_trials = hyperparameters['hpo_params']['num_trials']
+        if scheduler_options is None:
+            scheduler_options = hyperparameters['hpo_params']['scheduler_options']
+        if search_strategy.endswith('hyperband'):
+            # Specific defaults for hyperband scheduling
+            if scheduler_options is None:
```

Comment:
We may move the logic outside of `if search_strategy.endswith('hyperband'):` so that it will be an empty dict.

Reply:
I see that the logic is added in `.fit()` so I marked it as resolve.